README.md

```diff
@@ -97,8 +72,7 @@ The following decoded JSON web token
 **Payload**
 ```javascript
 {
-  "ver": "0.1.0",
-  "type": "embed",
```

Comment:
Since we removed dev/provision tokens we are only ever assuming "embed" type? Therefore removing the type param all together.  Is this correct?